pkg/drivers/hyperv/powershell_windows.go

```diff
@@ -2,40 +2,23 @@ package hyperv
 
 import (
 	"bufio"
-	"bytes"
 	"errors"
-	"os/exec"
-	"strings"
-
 	"fmt"
+	"strings"
 
 	log "github.com/code-ready/crc/pkg/crc/logging"
+	"github.com/code-ready/crc/pkg/os/windows/powershell"
 )
 
-var powershell string
-
 var (
 	ErrPowerShellNotFound = errors.New("Powershell was not found in the path")
 	ErrNotAdministrator   = errors.New("Hyper-v commands have to be run as an Administrator")
 	ErrNotInstalled       = errors.New("Hyper-V PowerShell Module is not available")
 )
 
-func init() {
-	powershell, _ = exec.LookPath("powershell.exe")
-}
-
 func cmdOut(args ...string) (string, error) {
-	args = append([]string{"-NoProfile", "-NonInteractive"}, args...)
-	cmd := exec.Command(powershell, args...)
-	log.Debugf("[executing ==>] : %v %v", powershell, strings.Join(args, " "))
-	var stdout bytes.Buffer
-	var stderr bytes.Buffer
-	cmd.Stdout = &stdout
-	cmd.Stderr = &stderr
-	err := cmd.Run()
-	log.Debugf("[stdout =====>] : %s", stdout.String())
-	log.Debugf("[stderr =====>] : %s", stderr.String())
-	return stdout.String(), err
+	stdout, _, err := powershell.Execute(args...)
```

Comment:
Just wondering before looking further; is everything run with Default Locale?

Reply:
I dont see mention of a locale.